pgmpy/estimators/EM.py

```diff
@@ -50,11 +51,14 @@ class ExpectationMaximization(ParameterEstimator):
     """
 
     def __init__(self, model, data, **kwargs):
-        if not isinstance(model, BayesianNetwork):
+        if not isinstance(model, (DAG, BayesianNetwork)):
             raise NotImplementedError(
-                "Expectation Maximization is only implemented for BayesianNetwork"
+                "Expectation Maximization is only implemented for DAG or BayesianNetwork"
             )
 
+        if isinstance(model, DAG):
+            model = BayesianNetwork(model.edges())
```

Comment:
This would miss diconnected nodes in the graph. Will need to add another line
```
model.add_nodes_from(model.nodes())
```

Reply:
Should i even add `latents` or is that taken care of?

Reply:
Good point, you would need to add latents as well.